x-pack/test/functional/apps/lens/persistent_context.ts

```diff
@@ -8,12 +8,24 @@ import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
-  const PageObjects = getPageObjects(['visualize', 'lens', 'header', 'timePicker']);
+  const PageObjects = getPageObjects(['visualize', 'lens', 'header', 'timePicker', 'common']);
```

Comment:
It doesn't look like `common` is used here

Reply:
fixed

Reply:
Appears to still be here? `common`

Reply:
resolved.